modules/tinymce/src/core/main/ts/api/EditorCommands.ts

```diff
@@ -11,8 +11,8 @@ import Editor from './Editor';
  * @class tinymce.EditorCommands
  */
 
-export type EditorCommandCallback<S> = (this: S, ui: boolean, value: any) => void;
-export type EditorCommandsCallback = (command: string, ui: boolean, value?: any) => void;
+export type EditorCommandCallback<S> = (this: S, ui: boolean, value: any, args?: ExecCommandArgs) => void;
+export type EditorCommandsCallback = (command: string, ui: boolean, value: any, args?: ExecCommandArgs) => void;
```

Comment:
Can't we keep the optional value?
```suggestion
export type EditorCommandsCallback = (command: string, ui: boolean, value?: any, args?: ExecCommandArgs) => void;
```